.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: xenial
+language: python
+python: -"3.7"
+
+# install the dependencies
+install:
+  - pip install -r requirements.txt
+
+# run the tests
+script:
+  - pylama
+  - isort -rc --check-only
+  - mypy --check-untyped-defs
```

Comment:
Please set --check-untyped-defs in mypy config itself and do not put anything here.

Reply:
@rakaar Implement this please.